lib/optimize.js

```diff
@@ -0,0 +1,87 @@
+'use strict'
+
+const serializerReturnRE = /^return ([a-zA-Z0-9]+)\(\w+\)$/
```

Comment:
How do you write this code?
Do you follow some automation?

Only AI will be able to read it tho

Reply:
Manually. Writing my unit test, then adapt to what I prefer.